packages/amplify-category-auth/src/provider-utils/supported-services.ts

```diff
@@ -1260,13 +1260,10 @@ const supportedServices = {
 export const getSupportedServices = (): any => {
   const keyToRemove = FeatureFlags.getBoolean('auth.forceAliasAttributes') ? 'usernameAttributes' : 'aliasAttributes';
   const inputs = supportedServices.Cognito.inputs.filter(input => input.key !== keyToRemove);
-
-  const cfnFilename = FeatureFlags.getBoolean('auth.forceAliasAttributes') ? 'auth-template-force-alias.yml.ejs' : 'auth-template.yml.ejs';
   return {
     Cognito: {
       ...supportedServices.Cognito,
       inputs,
```

Comment:
this might read a bit better as
```
supportedServices.Cognito.inputs = inputs;
return supportedServices;
```

However, that does actually modify supportedServices so if we don't want that, then you can keep what you have

Reply:
I originally did this and it became more difficult to test, since the test runs in a single process and the modified supportedServices object broke expectations.